vscode/src/common.ts

```diff
@@ -88,7 +88,7 @@ export const SUPPORTED_LANGUAGE_IDS = ["ruby", "erb"];
 export const FEATURE_FLAGS = {
   tapiocaAddon: 1.0,
   launcher: 0.1,
-  fullTestDiscovery: 0.05,
+  fullTestDiscovery: 0.15,
```

Comment:
Should we be more aggressive or are we concerned by the amount of support / bug report this may create?